### ISSUE TITLE
Topo with missing VM won't load

### ISSUE THREAD

**Issue by brentstewart** [GNS3/gns3-gui#2575]
**TLDR**: Topologies that include missing VMs can't be loaded
**Suggestion**: Replace missing VMs with dummy icon and load topology

**Long version**:  I run a remote GNS3 server, but had a topology that included a local VM running on Mint/VMWare Workstation.  I completed that work and deleted the local VM, but later decided to review the GNS3 project.  When I try to open the file, I get the following error:

VMware VM "X-1" [a283809d-5f12-4fe7-b276-9b570282c65f]: could not find VMX file "/home/brent/vmware/X/X.vmx"

Would it be possible to add an error check?  If this condition occurs, I recommend showing the same error, but then continuing to load the file.  Replace the missing device with a dummy icon labeled "_name_ could not be loaded."

You guys are doing great work!  This bug isn't a big deal for me and can be low priority - just didn't want to see it bite someone else down the line!

**Comment by grossmj** [comment]
https://gns3.com/community/discussion/vm-s-in-project-moved

**Comment by jean-christophe-manciot** [comment]
This feature request is definitely a must-have: an easier suggestion would be to load the project as-is without the node(s) using the missing image(s), and without all the related connections as well.

If we cannot retrieve the missing image(s), we won't ever be able to load the project, unless we rummage around in the .gns3 definition file.

**Comment by jean-christophe-manciot** [comment]
Another more evolved suggestion would be to:
- offer to replace the missing image with an existing one, using the preferences (RAM, vCPU, network, icon, ...) already defined for the corresponding node(s) 
- the UUID would be different, so there is no need to delete the contents of the project files for corresponding missing image(s).

**Comment by ClassicD123** [comment]
Im not sure if you are still having issues, however i simply created a generic txt file and changed the extension. 

viola.... :)

**Comment by alidhehibi** [comment]
Dear team

Anyone can help me with this problem, the VMX file was deleted from my laptop,

I need to reopen the GNS3 project without the missed VMX file,

Thank you

**Comment by josephmhiggins** [comment]
@ alidhehibi
The gns3 community, especially me, are better suited to help you with an accidental deletion of important files.
Open up a post on the gns3 community board and just list your
operating system name and version, and
your vmware workstation, or player, version.
